src/components/app/app.test.ts

```diff
@@ -189,7 +189,7 @@ describe('app test suite', () => {
     it('should store a session in a signed cookie', async () => {
       const response = await request(app).get('/test');
 
-      expect(response.header['set-cookie'][0]).toContain('pazmin-session.sig');
```

Comment:
Travis is 💢about this. Maybe `expect(response.header['set-cookie'].some(x => x.include('pazmin-session.sig'))).toBe(true)`